src/lib.rs

```diff
@@ -319,7 +320,19 @@ impl<B: MysqlShim<W>, R: Read, W: Write> MysqlIntermediary<B, R, W> {
                                 w.completed(0, 0)?;
                             }
                         }
+                    } else if q.starts_with(b"USE ") || q.starts_with(b"use ") {
+                        let w = InitWriter {
+                            writer: &mut self.writer,
+                        };
+                        let qstr = ::std::str::from_utf8(q)
+                                .map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))?;
+                        let re = Regex::new(r"^(?i)USE (?P<schema>[^@\s]+);?")
```

Comment:
I don't think we need a regular expression here. This will do:
```rust
let schema = q[3..].trim().strip_suffix(';');
```
That still doesn't handle quoting, but is probably fine for now.

Reply:
Perhaps we could also `.replace("`", "")`?